gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/FsSpecConsumer.java

```diff
@@ -55,10 +56,13 @@
   private Map<URI, Path> specToPathMap = new HashMap<>();
 
 
-  public FsSpecConsumer(Config config) {
+  public FsSpecConsumer(FileSystem fs, Config config) {
```

Comment:
Since it is a public API, should we leave an option of `FsSpecConsumer(Config config)`  (by calling this constructor)?